akka-stream/src/main/boilerplate/akka/stream/scaladsl/GraphApply.scala.template

```diff
@@ -50,6 +50,22 @@ trait GraphApply {
 
   ]
 
+  /**
+   * Creates a new [[Graph]] by importing the given graph list `graphs` and passing their [[Shape]]s
+   * along with the [[GraphDSL.Builder]] to the given create function.
+   */
+  def create[S <: Shape, IS <: Shape, Mat](graphs: Seq[Graph[IS, Mat]])(buildBlock: GraphDSL.Builder[Seq[Mat]] ⇒ Seq[IS] ⇒ S): Graph[S, Seq[Mat]] = {
```

Comment:
that should be `immutable.Seq` (several). We normally

```
import scala.collection.immutable
```

at the top of the file and then use that as `immutable.Seq`